cmd/juju/application/addunit.go

```diff
@@ -51,7 +51,7 @@ Add two units of mysql to existing machines 3 and 4:
 
    juju add-unit mysql -n 2 --to 3,4
 
-Add three units of mysql, one to machine 3 and the others to new
+Add three units of mysql, one to machine 7 and the others to new
```

Comment:
This should not be merged into develop! I have doscivered when I wanted to forward port my change that instead of changing the text, the example was adjusted on develop long before my fix landed in 2.4. Please remove it - text should be left as is here.

Reply:
👍